docs/environment-variables.md

```diff
@@ -96,26 +100,65 @@ Default: No upper bound, [TTL from IPNS Record](https://specs.ipfs.tech/ipns/ipn
 
 A comma-separated list of [multiaddresses](https://docs.libp2p.io/concepts/fundamentals/addressing/) of peers to stay connected to.
 
-
-If `RAINBOW_SEED` is set and `/p2p/rainbow-seed/N` value is found here, Rainbow
-will replace it with a valid `/p2p/` for a peer ID generated from same seed
-and index `N`.
+> [!TIP]
+> If `RAINBOW_SEED` is set and `/p2p/rainbow-seed/N` value is found here, Rainbow
+> will replace it with a valid `/p2p/` for a peer ID generated from same seed
+> and index `N`. This is useful when `RAINBOW_SEED_PEERING` can't be used,
+> or when peer routing should be skipped and specific address should be used.
 
 Default: not set (no peering)
 
+### `RAINBOW_SEED`
+
+Base58 seed to derive PeerID from. Can be generated with `rainbow gen-seed`.
+If set, requires `RAINBOW_SEED_INDEX` to be set as well.
+
+Default: not set
+
+### `RAINBOW_SEED_INDEX`
+
+Index to derivate the PeerID identity from `RAINBOW_SEED`.
+
+Default: not set
+
+### `RAINBOW_SEED_PEERING`
+
+> [!WARNING]
+> Experimental feature.
+
+Automated version of `RAINBOW_PEERING` which does not require providing multiaddrs.
+
+Instead, it will set up peering with peers that share the same seed (requires `RAINBOW_SEED_INDEX` to be set up).
+
+> [!NOTE]
+> Runs a separate light DHT for peer routing with the main host if DHT routing is disabled.
```

Comment:
What do `light DHT` and `main host` mean?

Is the light DHT a DHT instance that will only publish the peer records of the rainbow instance?

Reply:
By default, Rainbow runs with two DHT hosts:
- The "main host" which gives the identity of the Rainbow peer.
- The "dht host" that is only used for DHT lookups.

I don't have the full context on why this was decided but maybe @aschmahmann can give more information (he made https://github.com/ipfs/rainbow/pull/5 after all).

---

If the DHT shared host is enabled, the main host is used for the DHT.

---

To find peers using the seed-based peering, we need the DHT. This DHT instance must run with the main host, since this is the one that really identifies the Rainbow peer. If DHT shared host is enabled, we can just use that DHT. Otherwise, we need to run a DHT instance in parallel with the DHT host just for peer routing.